cmd/nerdctl/container_run_network_linux_test.go

```diff
@@ -245,7 +246,11 @@ func TestUniqueHostPortAssignement(t *testing.T) {
 			testContainerName1 := fmt.Sprintf("%s-%d-1", tID, i)
 			testContainerName2 := fmt.Sprintf("%s-%d-2", tID, i)
 			base := testutil.NewBase(t)
-			defer base.Cmd("rm", "-f", testContainerName1, testContainerName2).Run()
+			defer func() {
+				base.Cmd("rm", "-f", testContainerName1, testContainerName2).AssertOK()
+				// sleep is expected to deflake test-integration-docker-compatibility https://github.com/containerd/nerdctl/issues/2206
+				time.Sleep(3 * time.Second)
```

Comment:
I curios to know how this impact the test. Do you have any explanation @AkihiroSuda  ?

Reply:
No, my assumption was wrong 😞